RLBotPack/Codename_Cryo/src/GetBoost.py

```diff
@@ -82,7 +82,7 @@ def closest_full_boost(self, agent):
                     (boost.location[1] < agent.ball.location[1] and agent.team == 1)):
                 continue
             if abs(boost.location[0]) < 3000:
-                print("wrong boost, skipping")
+                print("@Tarehart there are ghost boosts!")
```

Comment:
Integrate with the discord API for max effectiveness haha

Reply:
Yeah i don't know if it's my fault but my bots see ghost boosts.